linuxkit/aws_test.yml

```diff
@@ -0,0 +1,36 @@
+kernel:
```

Comment:
I would modify `aws.yml` if you are just adding more config/containers.
When everything is working, we can generate more simplified files.

Reply:
Ok, this was just another file to troubleshoot. Will add SSH support